packages/commerce-sdk-react/package.json

```diff
@@ -32,7 +32,7 @@
     "test:watch": "npm test -- --watch",
     "test:inspect": "node --inspect-brk jest --runInBand",
     "test:no-mocked-responses": "cross-env NOCK_BACK_MODE=wild npm run test",
-    "test:update-mocked-responses": "cross-env NOCK_BACK_MODE=update npm test",
+    "test:update-mocked-responses": "cross-env NOCK_BACK_MODE=update internal-lib-build test",
```

Comment:
We shouldn't need to rebuilding before running test. 
We already build in a step by running `npm ci` in CI.

Reply:
@adamraya I'm confused the `internal-lib-build` is a node process that accepts a parameter (in this case a command called `test`). The reason for this change is the way that the terminal session was spawning a separate / new process which didn't show the output from `npm test` ... we were discussing this while debugging yesterday

Reply:
So, to put it more directly, my understanding is that this change is simply running `npm test` but doing so in the same Shell session

Reply:
Ignore my comment. I took a quick look at the PR on a small screen and saw the NPM command `build test`, but it's because my small screen is breaking the command into a new line. 
<img width="1379" alt="Screen Shot 2022-10-20 at 8 12 30 AM" src="https://user-images.githubusercontent.com/6024102/196988261-9c49b135-656a-40e8-a38f-6407522c7b9e.png">